Calling/Package.appxmanifest

```diff
@@ -1,50 +1,60 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-
-<Package
-  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
-  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
-  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
-  IgnorableNamespaces="uap mp">
-
-  <Identity
-    Name="83710438-7404-4fdd-a4f2-b207642c9b06"
-    Publisher="CN=xumo1"
-    Version="1.0.0.0" />
-
-  <mp:PhoneIdentity PhoneProductId="83710438-7404-4fdd-a4f2-b207642c9b06" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
-
-  <Properties>
-    <DisplayName>CallingQuickstart</DisplayName>
-    <PublisherDisplayName>xumo1</PublisherDisplayName>
-    <Logo>Assets\StoreLogo.png</Logo>
-  </Properties>
-
-  <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
-  </Dependencies>
-
-  <Resources>
-    <Resource Language="x-generate"/>
-  </Resources>
-
-  <Applications>
-    <Application Id="App"
-      Executable="$targetnametoken$.exe"
-      EntryPoint="CallingQuickstart.App">
-      <uap:VisualElements
-        DisplayName="CallingQuickstart"
-        Square150x150Logo="Assets\Square150x150Logo.png"
-        Square44x44Logo="Assets\Square44x44Logo.png"
-        Description="CallingQuickstart"
-        BackgroundColor="transparent">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png"/>
-        <uap:SplashScreen Image="Assets\SplashScreen.png" />
-      </uap:VisualElements>
-    </Application>
-  </Applications>
-
-  <Capabilities>
-    <Capability Name="internetClientServer"/>
-    <DeviceCapability Name="microphone"/>
-  </Capabilities>
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="646c0233-13f7-4ea3-b02e-703b86c51a27"
+    Publisher="CN=jowang"
```

Comment:
I don't know what's the common suggestion here, but maybe we can consider:
```suggestion
    Name="646c0233-13f7-4ea3-b02e-703b86c51a27"
    Publisher="CN=Microsoft"
```

Reply:
Will update, thanks.